src/modules/45-dashboards/pages/DashboardsContext.tsx

```diff
@@ -6,27 +6,42 @@
  */
 
 import React, { useState, useContext } from 'react'
+import { useParams } from 'react-router-dom'
 import type { Breadcrumb } from '@harness/uicore'
+import type { AccountPathProps } from '@common/interfaces/RouteInterfaces'
+import { PermissionIdentifier } from '@rbac/interfaces/PermissionIdentifier'
+import { FolderModel, useGetFolders } from 'services/custom-dashboards'
 
 export interface DashboardsContextProps {
   breadcrumbs: Breadcrumb[]
   includeBreadcrumbs: (breadcrumbs: Breadcrumb[]) => void
+  editableFolders?: FolderModel[]
```

Comment:
From the looks of your code, this will never be optional.
```suggestion
  editableFolders: FolderModel[]
```